.gitignore

```diff
@@ -11,6 +11,7 @@
 
 # misc
 .DS_Store
+.env
```

Comment:
This doesn't belong in .env. Is there a reason why you're trying to hide .env file? If there's settings that shouldn't be open to the user on the client, it should go into the back-end